applications/liberland.md

```diff
@@ -0,0 +1,241 @@
+- **Contact Email:** justice@liberland.org
+- **Website:** https://liberland.org
+
+### Legal Structure
+
+- **Registered Address:** 14/F, Manning House, 38-48 Queen’s Road Central, Central, Hong Kong
+- **Registered Legal Entity:** Liberland LLC
+
+### Team's experience
+
+Our team has worked on several substrate based projects such as Picasso and hashed-chain. The team has demonstrated knowledge of the substrate ecosystem and now we aim to build the first on-chain constitution and nation state on a substrate based chain.
+
+The team has started to develop the liberland chain on github:
+https://github.com/liberland/liberland_substrate
+
+
+### Team LinkedIn Profiles 
+
+- https://www.linkedin.com/in/balazs
+- https://www.linkedin.com/in/djstern
+- https://se.linkedin.com/in/filip-k%C3%A4lebo-
+
+## Development Status 
+
+We have already started to develop our chain, links:
+- https://github.com/liberland/liberland_substrate | Our current chain repository
+- https://testnet.liberland.org | Our public frontend with support for the polkadot.js wallet
+
+
+## Development Roadmap :nut_and_bolt:
+
+We aim to have a production ready system in 5 months, our chain will have the functionality to serve 
+
+
+### Overview
+
+- **Total Estimated Duration:** 5 months
+- **Full-Time Equivalent (FTE):**  3 FTE
+- **Total Costs:** 50 000 USD in BTC
+
+We also aim to enrolled in substrate's builders program.
+
+### Milestone 1: Citizens, LLM and Public Veto
+
+- **Estimated duration:** 2 month
+- **FTE:**  3
+- **Costs:** 20 000 USD
+
+The first milestone aims to provide all citizens with the functionality to have different roles depending on the account, take part of voting for elections and law proposals. We are also implementing the LLM(Liberland Merit) based on a predefined inflation curve that can be found here:
+https://docs.google.com/spreadsheets/d/1Q5DnVI4WuQWncMY985xuQ30HB7-bJz6X6EZFr2Vzr38/edit
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  MIT |
+| 0b. | Documentation | We will provide both inline code documentation as well as user guides that shows how people can interact with our pallets and chain |
+| 0c. | Testing Guide | We will have test files for all mayor functions, more about our testing procedure can be found under the testing section  |
+| 0d. | Docker | We will provide a Dockerfiles that can be used to run our node and also test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish a detailed article, walking a user through all functionality and features.)
+| 1. | Substrate chain | We will customize a substrate based code base and add our custom pallets |  
+| 2. | LLM module | We will create substrate pallet to handle the distribution and inflation of the LLM currency |  
+| 3. | Public Veto Module | We want our users to utilize our chain to have election and vote using a pallet  |  
```

Comment:
Did you consider reusing the [democracy pallet](https://crates.io/crates/pallet-democracy/3.0.0/dependencies) instead? If not, how do they differ? In general, it'd be great to have more information about what kind of logic this pallet will encapsulate.

Reply:
@takahser touched on this in the "Voting" section above, we want to use the vote functionality from democracy pallet and then add custom logic to allow certain users to vote on certain things